<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>700</height>
   </rect>
  </property>
    <property name="windowTitle">
     <string>zPrint</string>
    </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
         <widget class="QWidget" name="topBar1">
            <property name="objectName">
             <string>topBar1</string>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="topBar1Layout">
      <item>
       <widget class="QPushButton" name="btnThemeToggle">
        <property name="text">
         <string>Theme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReload">
        <property name="text">
         <string>Reload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnImport">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddModel">
        <property name="text">
         <string>Add Model</string>
        </property>
       </widget>
      </item>
            </layout>
         </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="topBar2Layout">
      <item>
       <widget class="QLineEdit" name="searchBox">
        <property name="placeholderText">
         <string>Search...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sortDropdown">
        <item>
         <property name="text">
          <string>Last viewed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Last created</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Name A-Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Name Z-A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Print Time</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="filterDropdown">
        <item>
         <property name="text">
          <string>All Materials</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>719</width>
         <height>616</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="galleryLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
